packages/react/src/button/Button.stories.tsx

```diff
@@ -1,11 +1,10 @@
-import { ComponentMeta, ComponentStory } from '@storybook/react';
+import { Meta, StoryObj } from '@storybook/react';
 import { allIcons, AvatarIcon } from '../icon';
 import { Button, ButtonLink } from './Button';
 
-export default {
+const meta: Meta<typeof Button> = {
```

Comment:
This change looks fine, but I can't see anything to do with animation?

Reply:
As part of the animation change, I had to update snapshots for Button. I did this while I was here :)